marklogic-data-hub/src/main/java/com/marklogic/hub/impl/MappingManagerImpl.java

```diff
@@ -47,18 +46,20 @@
 @Component
```

Comment:
If we're not autowiring this anymore, I think the Component annotation should be removed.